fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/SchemaInfoObject.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.database.utils.common;
+
+
+/**
+ * A database object (table, index, view etc) existing within a schema
+ */
+public class SchemaInfoObject {
+    public static enum Type {
+        INDEX,
+        PROCEDURE,
+        SEQUENCE,
+        TABLE,
+        VIEW
+    }
```

Comment:
Confirmation question... 

SQL Function and Procedure map to PROCEDURE above.

Reply:
Doesn't really matter as currently we're not listing procedures and functions although we could at some point